### PR TITLE
Chore/processing of depreciations and updating code to php72 minimum

### PR DESCRIPTION
**Pull request by ColonelMoutarde**
Changes proposed in this pull request:
- processing of depreciation and updating code to php72 minimum

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: app/Controllers/importExportController.php [hunk 1]
```diff
@@ -480,15 +480,15 @@ private function importJson(string $article_file, bool $starred = false): bool {
...
-				$knownLabels[$labelName]['articles'][] = array(
-						//'id' => $entry->id(),	//ID changes after commitNewEntries()
-						'id_feed' => $entry->feedId(),
-						'guid' => $entry->guid(),
-					);
+				$knownLabels[$labelName]['articles'][] = [
+		//'id' => $entry->id(),	//ID changes after commitNewEntries()
+		'id_feed' => $entry->feedId(),
+		'guid' => $entry->guid(),
+	];
```
Comment (Alkarex) [inline app/Controllers/importExportController.php]: Wrong indenting here

File: app/Controllers/entryController.php [hunk 2]
```diff
@@ -117,11 +117,11 @@ public function readAction(): void {
 
 		if (!$this->ajax) {
 			Minz_Request::good($is_read ? _t('feedback.sub.articles.marked_read') : _t('feedback.sub.articles.marked_unread'),
-			array(
+			[
 				'c' => 'index',
 				'a' => 'index',
 				'params' => $params,
-			));
+				]);
```
Comment (Alkarex) [inline app/Controllers/entryController.php]: Strange indenting of the closing part of the array in many of your changes here and below

File: app/install.php [hunk 3]
```diff
@@ -268,11 +268,7 @@ function checkStep0(): array {
 	$language = Minz_Session::param('language') != '' && in_array(Minz_Session::param('language'), $languages, true);
 	$sessionWorking = Minz_Session::param('sessionWorking') === 'ok';
 
-	return array(
-		'language' => $language ? 'ok' : 'ko',
-		'sessionWorking' => $sessionWorking ? 'ok' : 'ko',
-		'all' => $language && $sessionWorking ? 'ok' : 'ko'
-	);
+	return ['language' => $language ? 'ok' : 'ko', 'sessionWorking' => $sessionWorking ? 'ok' : 'ko', 'all' => $language && $sessionWorking ? 'ok' : 'ko'];
```
Comment (Alkarex) [inline app/install.php]: I could better like the multiline version for this part

File: app/SQL/install.sql.sqlite.php [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?php
-$GLOBALS['SQL_CREATE_DB'] = <<<'SQL'
+$GLOBALS['SQL_CREATE_DB'] = <<<'SQL_WRAP'
```
Comment (Alkarex) [inline app/SQL/install.sql.sqlite.php]: Why WRAP?
`SQL` specifically allows third-party language support for some IDEs

File: app/Models/FormAuth.php [hunk 5]
```diff
@@ -71,7 +71,7 @@ public static function deleteCookie(): void {
 			@unlink(DATA_PATH . '/tokens/' . $token . '.txt');
 		}
 
-		if (rand(0, 10) === 1) {
+		if (random_int(0, 10) === 1) {
```
Comment (Alkarex) [inline app/Models/FormAuth.php]: Waste of energy: no need to have an expensive RNG here

File: app/Models/FeedDAO.php [hunk 6]
```diff
@@ -1,16 +1,17 @@
...
+{
 
 	protected function addColumn(string $name): bool {
 		if ($this->pdo->inTransaction()) {
 			$this->pdo->commit();
 		}
 		Minz_Log::warning(__method__ . ': ' . $name);
 		try {
-			if ($name === 'kind') {	//v1.20.0
+			if ($name === 'kind') {    //v1.20.0
```
Comment (Alkarex) [inline app/Models/FeedDAO.php]: Please keep the whitespace as it was for such minor things (several similar cases below)

File: app/Models/Feed.php [hunk 7]
```diff
@@ -546,14 +546,30 @@ public function loadEntries(SimplePie $simplePie): Traversable {
...
+						}
+						if ($length != '') {
+							$attributeEnclosure['length'] = intval($length);
+						}
+						if ($height != '') {
+							$attributeEnclosure['height'] = intval($height);
+						}
+						if ($width != '') {
+							$attributeEnclosure['width'] = intval($width);
+						}
```
Comment (Alkarex) [inline app/Models/Feed.php:572]: Mja, not a fan for this specific case, but no big deal

File: app/Models/FormAuth.php (hunk 5 above)
Comment (ColonelMoutarde) [reply app/Models/FormAuth.php]: https://stackoverflow.com/questions/44228718/php-rand-vs-random-int
ok but i dislike rand()

Comment (Alkarex) [reply app/Models/FormAuth.php]: What is your issue with `rand()`? Your link indicates that it is the best one to use there ;-)
A Mersenne Twister RNG is very fine here

File: app/Controllers/entryController.php (hunk 2 above)
Comment (Alkarex) [reply app/Controllers/entryController.php]: There were still many arrays with strange indenting. Fixed in https://github.com/FreshRSS/FreshRSS/pull/5504/commits/a13685f4ed20537e4fccd4b603ee1e45d9e516ce

File: app/Controllers/feedController.php [hunk 8]
```diff
@@ -722,13 +720,18 @@ public function actualizeAction(): int {
 		return $updated_feeds;
 	}
 
+	/**
+	 * @throws Minz_ConfigurationNamespaceException
+	 * @throws JsonException
```
Comment (Alkarex) [inline app/Controllers/feedController.php:725]: This line (several instances) looks wrong. Fixed in https://github.com/FreshRSS/FreshRSS/pull/6037

**Pull request merged**